src/components/DeckPreview.js

```diff
@@ -8,7 +8,7 @@ const DeckPreview = ({ deck }) => {
   const maxLength = 35;
   const { date } = deck;
 
-  let { name } = deck;
+  let { name } = deck.seats[0];
```

Comment:
Not using `.username` or `.name` here? Or is `name` not a good variable name for `deck.seats[0]`?